addon/components/pix-checkbox.js

```diff
@@ -10,11 +9,6 @@ const labelSizeToClass = new Map([
 export default class PixCheckbox extends Component {
   constructor() {
     super(...arguments);
-
-    if (!this.args.label) {
-      throw new Error('ERROR in PixCheckbox component, you must provide @label params');
-    }
```

Comment:
**question**
Est-ce qu'en utilisant le `yield` plutôt qu'un argument pour le label, on ne perd pas le fait de le rendre obligatoire ? Est-ce qu'il y a un moyen de lancer une exception si je composant n'a pas de contenu à yielder ?

Reply:
j'ai juste trouvé le moyen d'afficher un message à la place du label pour dire que le yield est require pour avoir un label sur le composant. (le yield ne semble pas dispo dans le `this.`